### ISSUE TITLE
Export model as .pb file in android

### ISSUE THREAD

**Issue by adarshrana205** [google-research/bert#1091]
I have created a CNN model in jupyter notebook.But i want to export this model in android studio as .pb file. 
I used the following code:

def export_model(saver, model, input_node_names, output_node_name):
    tf.io.write_graph(tf.compat.v1.keras.backend.get_session().graph_def, 'out',MODEL_NAME + '_graph.pbtxt')

    saver.save(tf.compat.v1.keras.backend.get_session(),save_path, 'out/' + MODEL_NAME + '.chkp')

    freeze_graph.freeze_graph('out/' + MODEL_NAME + '_graph.pbtxt', None,False, 'out/' + MODEL_NAME + '.chkp', output_node_name,
        "save/restore_all", "save/Const:0",'out/frozen_' + MODEL_NAME + '.pb', True, "")

    input_graph_def = tf.GraphDef()
    with tf.gfile.Open('out/frozen_' + MODEL_NAME + '.pb', "rb") as f:
        input_graph_def.ParseFromString(f.read())

    output_graph_def = optimize_for_inference_lib.optimize_for_inference(
            input_graph_def, input_node_names, [output_node_name],
            tf.float32.as_datatype_enum)

    with tf.gfile.FastGFile('out/opt_' + MODEL_NAME + '.pb', "wb") as f:
        f.write(output_graph_def.SerializeToString())

    print("graph saved!")

 export_model(tf.compat.v1.train.Saver(), model, ["conv2d_1_input"], "dense_2/Softmax")
    

But this is giving me an error.
So please someone modify it so that i can use it in android.
